packages/blockchain/src/index.ts

```diff
@@ -369,8 +393,8 @@ export default class Blockchain implements BlockchainInterface {
       await block.validate(this)
     }
 
-    if (this._validatePow && this.ethash) {
-      const valid = await this.ethash.verifyPOW(block)
+    if (this._validateConsensus && this._ethash) {
```

Comment:
According to constructor condition there couldn't be `_validateConsensus` without `_ethash` instance, so the condition could be simplified to just check `_validateConsensus` to be true.

Reply:
Technically you are right, I just decided to keep this for now to limit a bit the risk that this check is forgotten once the library is expanded with other consensus types. I will put this check separately into the next line to make this more clear on next push.